### ISSUE TITLE
Types are not up to date

### ISSUE THREAD

**Issue by whazami** [pmndrs/postprocessing#519]
### Description

Everything is in the title. For instance the type of the BloomEffect's options in the `types/index.d.ts` file are:

```ts
export type BloomEffectOptions = {
	blendFunction?: BlendFunction;
	mipmapBlur?: boolean;
	luminanceThreshold?: number;
	luminanceSmoothing?: number;
	intensity?: number;
	resolutionScale?: number;
	resolutionX?: number;
	resolutionY?: number;
	width?: number;
	height?: number;
	kernelSize?: KernelSize;
};
```

and should be something like this:

```ts
export type BloomEffectOptions = {
	blendFunction?: BlendFunction;
	luminanceThreshold?: number;
	luminanceSmoothing?: number;
	mipmapBlur?: boolean;
	intensity?: number;
        radius?: number;
        levels?: number;
	kernelSize?: KernelSize;
	resolutionScale?: number;
	width?: number;
	height?: number;
	resolutionX?: number;
	resolutionY?: number;
};
```
### Library versions used

 - Three: 0.153.0
 - Post Processing: 6.32.2

**Comment by vanruesc** [comment]
Thanks for the report!

**Comment by vanruesc** [comment]
Fixed in `postprocessing@6.33.0`.
